src/main/java/org/zeroturnaround/javarebel/maven/util/SystemUtils.java

```diff
@@ -0,0 +1,70 @@
+package org.zeroturnaround.javarebel.maven.util;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
+/**
+ * Util class to ensure safe string representation for using in paths and URLs.
+ */
+public final class SystemUtils {
```

Comment:
As this particular piece of legacy code is duplicated now should we create an engineering task to make it into a library that could be publish publicly?

Reply:
Will be addressed in further engineering task JR-10262.